### PR TITLE
fixed EditProfile styling and avatar circle

### PR DESCRIPTION
**Pull request by andreadelrio**
Covers issue #89 and #91 
What was done:

- Fixed styling for EditProfile form (fields now use material-ui's [TextField](http://www.material-ui.com/#/components/text-field))

- Added sass version of Bootstrap to make use of its built-in media queries (in the future will refactor the rest of our css so that it uses them).

- Fixed squised avatar circle.

### CODE REVIEW

File: src/UniversalStyles.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
-@import 'https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-alpha.4/css/bootstrap.min.css';
+// @import 'https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-alpha.4/css/bootstrap.min.css';
```
Comment (davidascher) [inline src/UniversalStyles.scss]: Feel free to remove the old import.

File: src/components/EditProfile.scss [hunk 2]
```diff
@@ -112,22 +75,13 @@
...
-    display: block;
-    width: 100%;
-    box-sizing: border-box;
-    @media (max-width: 639px) {
-      font-size: 32px;
-      line-height: 36px;
-      padding: 16px 24px;
-    }
-  }
+  // .name-input {
```
Comment (davidascher) [inline src/components/EditProfile.scss]: also remove commented out stuff.

File: src/components/GeoSuggest.scss [hunk 3]
```diff
@@ -1,39 +1,56 @@
...
-  text-align: left;
-  @media (max-width: 639px) {
-    width: 100%;
-  }
-}
-.geosuggest__input {
-  border: none;
-  width: 98%;
-}
+// /**
```
Comment (davidascher) [inline src/components/GeoSuggest.scss]: same.

File: src/components/GeoSuggest.scss [hunk 4]
```diff
@@ -67,13 +48,11 @@
...
     padding-left: 0;
   }
-  // border: 2px solid red;
-  // box-sizing: border-box;
 }
 
-/**
- * A geosuggest item
- */
+// /**
```
Comment (davidascher) [inline src/components/GeoSuggest.scss]: you know what =)

**Comment by andreadelrio** [comment]
@davidascher just removed comments!

**Pull request merged**